guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -57,40 +61,48 @@ For more information on custom SSL certificates signed by a Certificate Authorit
 endif::[]
 ifdef::katello[]
 . Copy the resulting tarball to your {SmartProxy}, for this example we will use `/root/myproxy.example.com-certs.tar`
+endif::[]
+ifdef::foreman-el,katello[]
 . Update repositories
 +
-.For {EL} 7 Users:
 [options="nowrap" subs="attributes"]
 ----
-# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm \
-                https://yum.theforeman.org/katello/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
-----
-+
-.For {EL} 8 Users:
-[options="nowrap" subs="attributes"]
-----
-# dnf update -y https://yum.theforeman.org/releases/{ProjectVersion}/el8/x86_64/foreman-release.rpm \
-                https://yum.theforeman.org/katello/{KatelloVersion}/katello/el8/x86_64/katello-repos-latest.rpm
+ifdef::katello[]
+# {package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el8/x86_64/foreman-release.rpm \
+  https://yum.theforeman.org/katello/{KatelloVersion}/katello/el8/x86_64/katello-repos-latest.rpm
+endif::[]
+ifdef::foreman-el[]
+# {package-update} https://yum.theforeman.org/releases/{ProjectVersion}/el8/x86_64/foreman-release.rpm
+endif::[]
 ----
-. Ensure the module streams are enabled for {EL} 8:
+. Ensure the module streams are enabled:
 +
 [options="nowrap" subs="attributes"]
 ----
-# dnf module enable -y katello:el8 pulpcore:el8
+ifdef::katello[]
+# dnf module enable katello:el8 pulpcore:el8
+endif::[]
+ifdef::foreman-el[]
+# dnf module enable foreman:el8
+endif::[]
 ----
-. Clean the yum cache and update the required packages:
+. Update the required packages:
 +
 [options="nowrap" subs="attributes"]
 ----
-# yum clean all
-# yum -y update
+# {package-update}
 ----
 +
 . Run the installer:
 +
 ----
+ifdef::katello[]
 # foreman-installer --certs-tar-file /root/myproxy.example.com-certs.tar \
                     --certs-update-all --certs-regenerate true --certs-deploy true
+endif::[]
+ifndef::katello[]
+# foreman-installer
+endif::[]
```

Comment:
We should be able to do this action for Foreman and Katello. The lines 100-101 talking about certificates was needed only for one particular upgrade and is not required for the current upgrade.

Reply:
Given I want to cherry pick this to 3.4, this is also true for 3.3 -> 3.4, correct?